hazelcast/src/main/java/com/hazelcast/config/EvictionConfig.java

```diff
@@ -53,6 +52,7 @@
      */
     public static final EvictionPolicy DEFAULT_EVICTION_POLICY = EvictionPolicy.LRU;
 
+    protected boolean isSizeConfigured;
```

Comment:
I just added this field and an extra `EvictionConfigAccessor` class, so the interface of the configuration itself didn't change.